bigframes/operations/blob.py

```diff
@@ -37,7 +40,29 @@ def metadata(self):
         details_json = self._apply_unary_op(ops.obj_fetch_metadata_op).struct.field(
             "details"
         )
-
         import bigframes.bigquery as bbq
 
         return bbq.json_extract(details_json, "$.gcs_metadata")
+
+    def display(self, n: int = 3):
```

Comment:
can we call it peek() to have some semantic resemblance to the dataframe/series peek()?